### ISSUE TITLE
Accessibility error in InAndOutWidget

### ISSUE THREAD

**Issue by federicadelia** [plone/Products.CMFPlone#643]
`>>` or `<<` buttons of InAndOutWidget cannot be read clearly by all screen readers (NVDA only reads "button") and then the alternative text isn't enough.

**Comment by ale-rt** [comment]
No activity happened in this issue for years.
Please feel free to reopen it if it is still relevant for Plone 5.2 or Plone 6.0.
